### ISSUE TITLE
Ret pally issue...

### ISSUE THREAD

**Issue by Elfyau** [Toludin/Healium#16]
I dont know if it is just a Ret pally issue or pally issue in general, I use healium on my ret pally to provide a few detox's and rez's when needed but with healium on the game soft freezes every few seconds and totally freezes in dungeons

**Comment by Toludin** [comment]
Wow...that sounds awful. I haven't heard of this issue before. Is it happening all over the place (even if you aren't in a group)? I can test it on my Ret Pally. Also you might want to try an addon like http://www.wowinterface.com/downloads/info7362-AddonUsage.html and make sure it's Healium that's causing the problem (though I imagine you're not seeing the issue when Healium is off?)

**Comment by Elfyau** [comment]
yeah tested last night absolutely no issue at all once turned off

**Comment by Gal10** [comment]
Hi Elfyau,
am using healium on my holy paladin and resto druid with nil problems. I suspect you strill have an old version of healium in your interface/wtf files. suggest to go into twitch or curse client and delete the addon from there, cause if you still have it there it will be loaded every time you log in game and make mentioned issues in your game.

**Comment by Elfyau** [comment]
its not the holy paladin that has the issue it is the Ret pally I believe
it has something to do with Greater Blessing of Wisdom and the Blessing of
Protection spells. (I also tried your suggestion and it still happens, I
have no problem with any other spec or toon just the Ret Pally)

**Comment by Toludin** [comment]
I'm not able to reproduce this issue on my ret paladin (though I haven't tried in a dungeon yet) - but with Healium on out in the world running around, I'm getting no performance issues.

Can you take a screenshot of your setup and settings? Perhaps it's some combo there that's causing the issues.

**Comment by Gal10** [comment]
Changed specialization to retri paladin on my toon and all works fine. Do you use any macro/script on retribution and draged it to the healium frame? maybe this is what cause the problem.

**Comment by Elfyau** [comment]
doubled checked and I dont :(

**Comment by Gal10** [comment]
One more thing that can cause your problems -could be your addons memory capacity which is over loaded for your pc and causes all these freezes issues especially inside dungeons/raids.  Take off some other addons  and leave healium then try out dungeon again.

**Comment by Gal10** [comment]
Elfyau I suggest you download Bug sack+bug grabber addons, go dungeon and check which errors are shown to find out exact freezes reasons.
